documentation/source/user_section/tutorials/non-template-registration/contrast-agnostic-registration/before-starting.rst

```diff
@@ -0,0 +1,15 @@
+Before starting this tutorial
+#############################
+
+#. Make sure that you have the following files in your working directory:
+
+   * ``single_subject/data/t1/t1.nii.gz``: An anatomical spinal cord scan in the T1 contrast.
+   * ``single_subject/data/t2/t2.nii.gz``: An anatomical spinal cord scan in the T2 contrast.
+
+   You can get these files by downloading :sct_tutorial_data:`data_contrast-agnostic-registration.zip`.
```

Comment:
This PR corresponds with:

* https://github.com/spinalcordtoolbox/sct_tutorial_data/pull/11

Once that PR is merged, we will need to create a new `sct_tutorial_data` release, then update this line:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/50f030fa55609f4e1020705acc12161b51b4ba05/documentation/source/conf.py#L100

Reply:
Until the new release of `sct_tutorial_data`, the current link is 404 not found. Could we add direct links to [t1/t1.nii.gz](https://github.com/spinalcordtoolbox/sct_tutorial_data/raw/master/single_subject/data/t1/t1.nii.gz) and [t2/t2.nii.gz](https://github.com/spinalcordtoolbox/sct_tutorial_data/raw/master/single_subject/data/t2/t2.nii.gz) in the meantime?

Reply:
The 404 should only be temporary, I think? 

(For past tutorial contributions, making a new release for `sct_tutorial_data` is usually done quite quickly. So, I was thinking that we would just merge https://github.com/spinalcordtoolbox/sct_tutorial_data/pull/11 upon review, run the release-creation workflow, then have the dataset link updated all within the scope of this PR.)

But, I may be misunderstanding -- did you have a different order in mind? :slightly_smiling_face:

Reply:
Ah ok, as long as all of that can be done today, so we can make the 5.7 release tomorrow morning.

Reply:
This morning, I've done the following:

* Merged the above `sct_tutorial_data` PR.
* Created a new release via https://github.com/spinalcordtoolbox/sct_tutorial_data/runs/7559810584?check_suite_focus=true (currently in-progress at time of writing)
* Updated the dataset URL via [137ca73](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3831/commits/137ca73c3a3765a4fef33ba5c4b38ec5c251176a).

Reply:
Great, thanks! The build has finished, and I confirmed that the link works.